codefresh.yml

```
@@ -0,0 +1,74 @@
+# More examples of Codefresh YAML can be found at
+# https://codefresh.io/docs/docs/yaml-examples/examples/
+
+version: "1.0"
+# Stages can help you organize your steps in stages
+stages:
+  - "clone"
+  - "test"
+  - "release"
+
+steps:
+  clone:
+    title: "Cloning repository"
+    type: "git-clone"
+    repo: "a2i2/surround"
+    # CF_BRANCH value is auto set when pipeline is triggered
+    # Learn more at codefresh.io/docs/docs/codefresh-yaml/variables/
+    revision: "${{CF_BRANCH}}"
+    git: "github"
+    stage: "clone"
+
+  test:
+    title: "Run Python tests"
+    type: "freestyle" # Run any command
+    image: "python:${{PYTHON_VERSION}}" # The image in which command will be executed
+    working_directory: "${{clone}}" # Running command where code cloned
+    commands:
+      # Install sudo
+      - apt-get update && apt-get install sudo -y
+
+      # Install surround
+      - sudo python setup.py install
```

```suggestion
      - python3 setup.py install
```
It's generally a good idea to run `python2` or `python3` as opposed to `python` so that it's explicit which version of Python you're using. 

(It's a moot point in this instance since we're using a `python` Docker image, but it's easier to just do it everywhere for consistency).